lib/transport/abstract-transport.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var util = require("util");
+var EventEmitter = require('events');
```

Comment:
yeah it was added around 0.12.x, I'd say to be safe (since we do claim to support 0.10.x I guess) just stick with `require('events').EventEmitter`. It's an alias in the newer code anyway